### ISSUE TITLE
incorrect bbi init maxlim

### ISSUE THREAD

**Issue by dpastoor** [metrumresearchgroup/babylon#149]
I think this might be from the misspelling of json --> jason causing the parser of that to fail

https://github.com/metrumresearchgroup/babylon/blob/77260ddc73cbaa4cf8ea9290c0b943c9b28bd0ef/configlib/config.go#L62

currently this is the output

```yaml
mpi_exec_path: /usr/local/mpich3/bin/mpiexec
nmfe_options:
  MaxLim: 100
nonmem:
```

would expect

```diff

mpi_exec_path: /usr/local/mpich3/bin/mpiexec
nmfe_options:
-  MaxLim: 100
+  maxlim: 100
nonmem:
```

**Comment by shairozan** [comment]
Yeah that's what it is. After I updated the tag, the JSON shows:

```json
        "parallel": false,
        "JSON": false,
        "nmfe_options": {
            "maxlim": 100
        },
        "mpi_exec_path": "/usr/local/mpich3/bin/mpiexec",
        "parallel_timeout": 2147483647
    },
```

**Comment by shairozan** [comment]
Fixed in RC-3

**Comment by shairozan** [comment]
Released in v2.1.0
